course/layouts/index.coursedata.json

```diff
@@ -0,0 +1,27 @@
+{{- $courseData := .Site.Data.course -}}
+{{- $instructors := slice -}}
+{{- if not (eq  $courseData.instructors nil) -}}
+{{- $instructors = partial "get_instructors.html" $courseData.instructors.content -}}
+{{- end -}}
+
+{
+  "course_title": {{- $courseData.course_title | jsonify -}},
+  "course_description": {{- $courseData.course_description | jsonify -}},
+  "site_uid": {{- $courseData.site_uid | jsonify -}},
+  "legacy_uid": {{- $courseData.legacy_uid | jsonify -}},
+  "instructors": [
```

Comment:
This could be simplified using a combination of the Hugo [apply](https://gohugo.io/functions/apply/) and [index](https://gohugo.io/functions/index-function/) functions, no need to manually loop and delimit.  For example:

`"instructors": {{- apply $instructors "index" "." "title" | jsonify -}},`

Reply:
So actually, the import to open-discussions will be easier if we include the name parts separately in the output, and not just the full title. I made that update.